bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WJetsToLNu/W2JetsToLNu_13TeV-madgraphMLM-pythia8/W2JetsToLNu_13TeV-madgraphMLM-pythia8_run_card.dat

```diff
@@ -48,9 +48,8 @@
 #*********************************************************************
 # PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
 #*********************************************************************
- 'lhapdf'    = pdlabel     ! PDF set                                  
-$DEFAULT_PDF_SETS = lhaid
-$DEFAULT_PDF_MEMBERS = reweight_PDF     ! if pdlabel=lhapdf, this is the lhapdf number 
+ lhapdf = pdlabel ! PDF set
+ 306000 = lhaid
```

Comment:
I am confused now, why did you revert the lhaid?

Reply:
Hi Luca,

he did it according to the twiki for LO sample:

https://twiki.cern.ch/twiki/bin/viewauth/CMS/QuickGuideMadGraph5aMCatNLO#PDF_Choice_for_2017_production

Reply:
I am afraid you need to go to the second paragraph...
also cfr https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

Reply:
Agreed that the twiki is confusing. Actually this syntax is supported at LO as well. Updating now.

Reply:
T = pdfwgt this should also be included right?

Reply:
yes

Reply:
Thanks @efeyazgan , and @kdlong for changing the twiki, it is clear now.  Review them please and submit the PR.